### PR TITLE
Reexport endpoint methods

### PR DESCRIPTION
**Pull request by rofferom**
Reexport
* `Endpoint::close()`
* `Endpoint::reject_new_connections()`

### CODE REVIEW

**Comment by tausifcreates** [comment]
Hi, what does it achieve?

**Comment by rofferom** [comment]
I'd like to use quinn's endpoint helpers to close all active connections, and also have a quick way to forbid now incoming connections.

Quinn already provides all of this, I just reexport what I need.

File: wtransport/src/endpoint.rs [hunk 1]
```diff
@@ -130,6 +131,11 @@ impl<Side> Endpoint<Side> {
         Ok(socket)
     }
 
+    /// Close all of this endpoint's connections immediately and cease accepting new connections.
```
Comment (BiagioFesta) [inline wtransport/src/endpoint.rs:134]: Using rust-doc conventions:
`Close` -> `Closes`

File: wtransport/src/endpoint.rs [hunk 2]
```diff
@@ -199,6 +205,11 @@ impl Endpoint<endpoint_side::Server> {
 
         Ok(())
     }
+
+    /// Reject new incoming connections without affecting existing connections
```
Comment (BiagioFesta) [inline wtransport/src/endpoint.rs:209]: Using rust-doc conventions:
`Reject` -> `Rejects`

**Review by BiagioFesta — CHANGES_REQUESTED** [review]

File: wtransport/src/endpoint.rs (hunk 2 above)
Comment (rofferom) [reply wtransport/src/endpoint.rs]: Fixed

File: wtransport/src/endpoint.rs (hunk 1 above)
Comment (rofferom) [reply wtransport/src/endpoint.rs]: Fixed

**Review by BiagioFesta — APPROVED** [review]

**Comment by BiagioFesta** [comment]
Thanks!

**Pull request merged**